### ISSUE TITLE
Some problems

### ISSUE THREAD

**Issue by ghost** [haveric/Statues#4]
Hello,

I'm really proud of your plugin but i have some problems:

Here you can see my skin: http://imgur.com/5OiEN
- The Statue is missing the mask (http://imgur.com/X81RO)
- Some parts of the Statue are turned (http://imgur.com/T78ep)
- The Diamond Block keeps being there (http://imgur.com/QXeQD)
- The Statue is empty and the visitors get annoying monster sounds (http://imgur.com/bgatB)
- If I type the optional command part (http://imgur.com/fOq7s) nothing happens (http://imgur.com/IBPgo)

Hope you can fix them all :)

Relies upon: 
#5, #6, #7, #8, #9, #10

**Comment by haveric** [comment]
Thanks, I'm going to add individual issues for most of these and link them from here.  These are the main reasons I haven't released it on bukkitdev yet and with senior design, work, and homework, I haven't had a lot of time to work on it.

That command makes no sense and looking at the command I can kind of understand why you did that.  The name that is yellow is supposed to be who it is currently set to.  I'll try to make that clearer.

**Comment by haveric** [comment]
Also, do you have a plugin that is stripping color from text? It would make lot more sense and look cleaner if you didn't do that.

**Comment by ghost** [comment]
Do you mean like: "/statues lordtobi &4AWESOME!" ?
And the 'AWESOME!' turns into red when it is on the sign?

**Comment by haveric** [comment]
No, I mean if you do /statues haveric, and then look at /statues, it should say /statues {name} (haveric).  It is just there to tell you whose statue you are going to be building if you need to verify it.

**Comment by ghost** [comment]
Oh OK now I understand, but what do you want with colors?

**Comment by haveric** [comment]
I'm just confused as to why you don't have colors in your chat.  Most plugins (including mine) use color to make those commands easier to read and understand.  In the example above, /statues {name} (haveric), the haveric would be a Yellow/Gold color which would stick out as being separate from the command.

Is there a plugin you are using that is removing the color?

**Comment by ghost** [comment]
Try to replace "ChatColor msgColor = ChatColor.DARK_AQUA;" with "ChatColor.DARK_AQUA" in the message part of Commands.java

**Comment by haveric** [comment]
No, it works fine for me. I'm just wondering why your text, in this [screenshot](http://imgur.com/fOq7s) is all white.

**Comment by ghost** [comment]
This a problem of MY Java, I don't know why. But it's not your plugin ;)
